src/Rules.Framework/Core/Rule.cs

```diff
@@ -20,17 +22,17 @@ public class Rule<TContentType, TConditionType>
         public DateTime DateBegin { get; internal set; }
 
         /// <summary>
-        /// Gets the date from which the rule ceases to be applicable.
+        /// Gets and sets the date from which the rule ceases to be applicable.
         /// </summary>
-        public DateTime? DateEnd { get; internal set; }
+        public DateTime? DateEnd { get; set; }
 
         /// <summary>
         /// Gets the rule name.
         /// </summary>
         public string Name { get; internal set; }
 
         /// <summary>
-        /// Gets the rule priority compared to other rules (preferrably it is unique).
+        /// Gets and sets the rule priority compared to other rules (preferrably it is unique).
```

Comment:
typo: preferably (not yours but is in the same line :) )

Reply:
Done